gatherers/sandboxed-iframes.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gather = require('./gather');
+
+class SandboxedIframes extends Gather {
```

Comment:
unless that is you can come up with a case where that wold not be correct...

I an sorta see both sides however hmmm --- the html gather phase and then requerying the DOM would require us to create a vdom with the string.. which is boo.

HMMM open to ideas, this is not a blocker.